### DISCUSSION TITLE
Breaking change: remove the builtin autoprefixer ability of CSS

### DISCUSSION THREAD

**Discussion by hyf0** [web-infra-dev/rspack#3866] [Core] [answered]
Unreasonable designs resulting from the need to quickly support the business do not conform to the principles of the existing architecture design and should be removed.

Another purpose of this change is to align the related logic with Webpack.

Users would need to use `postcss-loader` and `autoprefixer` themselves just like how Webpack users doing now.

**Comment by chenjiahan** [comment]
Will Rspack provide migration guides for these breaking changes?

**Reply by hyf0** [reply]
Yes, we should.

**Comment by hyf0** [comment, marked as answer]
https://github.com/web-infra-dev/rspack/pull/3874
